app/subscriber/src/features/my-reports/edit/ReportEditForm.tsx

```diff
@@ -257,6 +257,7 @@ export const ReportEditForm = React.forwardRef<HTMLDivElement | null, IReportEdi
               if (instance) {
                 const result = await updateReportInstance({
                   ...instance,
+                  body: '',
```

Comment:
Empty the `report.instance.body` so that it can be regenerated when a report is unlocked.